src/display/display.js

```diff
@@ -14,12 +14,13 @@ export class Display {
       setBoard(board);
     }
   };
-  displayWinMessage = (response, toast) => {
+  displayWinMessage = (response, toast, game) => {
     if (response.win) {
       toast.success(response.win, {
         autoClose: false,
         hideProgressBar: true
       });
+      game.win = true;
```

Comment:
I feel like this method should only be called when game is won, so the additional check on `response.win` and setting `game.win` to `true` feels awkward to me. Could the method caller handles the responsibility of determining whether the game is won?